### PR TITLE
python plugin: support pbr/setup.cfg projects.

### PR DESCRIPTION
**Pull request by sergiusens**
Create the entry_points/console_scripts defined in setup.cfg
when using pbr in setup.py

LP: #1670852
Signed-off-by: Sergio Schvezov <sergio.schvezov@canonical.com>

### CODE REVIEW

File: integration_tests/test_python_plugin.py [hunk 1]
```diff
@@ -84,6 +87,48 @@ def test_build_rewrites_shebangs(self):
...
+            for python_dir in ['python2', 'python3']:
+                d = os.path.join(self.path, python_dir)
+                check_output(['git', 'init'], cwd=d)
+                check_output(
+                    ['git', 'config', 'user.name', 'Test User'], cwd=d)
+                check_output(
+                    ['git', 'config', 'user.email', '<test.user@example.com'],
+                    cwd=d)
+                check_output(['git', 'add', '.'], cwd=d)
+                check_output(['git', 'commit', '-m', 'initial'], cwd=d)
```
Comment (elopio) [inline integration_tests/test_python_plugin.py]: why not just running these steps, instead of putting them in a callable?

Comment (sergiusens) [inline integration_tests/test_python_plugin.py]: I didn't have a split API to get this done. Let's create a bug and get it done in the future. I sort of prefer this mechanism though.

Comment (sergiusens) [inline integration_tests/test_python_plugin.py]: iow, this is really a corner case scenario due to some limitations/features in git. If it is used often enough then maybe we can split, but it should be unlikely.

**Comment by elopio** [comment]
The integration tests fail because of this:
# PBR Generated from u'console_scripts'
# PBR Generated from 'console_scripts'
